charts/kube-prometheus-stack/hack/sync_prometheus_rules.py

```diff
@@ -35,7 +35,7 @@ def new_representer(dumper, data):
         'min_kubernetes': '1.14.0-0'
     },
     {
-        'source': 'https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/release-0.1/manifests/prometheus-rules.yaml',
+        'source': 'https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/master/manifests/prometheus-rules.yaml',
```

Comment:
@mrueg @bismarck please be aware that prometheus-operator has removed this file from its repo:
https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/master/manifests/prometheus-rules.yaml

As they've moved in this PR:
https://github.com/prometheus-operator/kube-prometheus/pull/873/files?file-filters%5B%5D=.libsonnet&file-filters%5B%5D=.yaml&hide-deleted-files=true#diff-1d254ebb0389617e83d5b134983121c1ffbee7f93e68fd80cf4f1d22d93fbc43